.gitignore

```diff
@@ -1,3 +1,10 @@
 /.idea/
 __pycache__
 .DS_Store
+
+# Devenv
+.envrc
```

Comment:
I don't want to enable automatic entry into devenv.sh as it requires quite a bit of setup locally.